<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataInputPanel</class>
 <widget class="QDialog" name="DataInputPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>431</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>尺寸(cm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="description_textEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="size_width_lineEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>290</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="size_length_lineEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="size_height_lineEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>290</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>540</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancle_pushButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>540</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>长</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>290</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>宽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>290</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>高</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>基本信息</string>
   </property>
   <widget class="QLineEdit" name="author_lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>作者</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="year_lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>年代</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="type_comboBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>基础</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>墙壁</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>柱子</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>楼板</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>房门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>窗户</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>屋顶</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>梁</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>其他</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="id_comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>手动编号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>自动编号</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>编号</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>结构信息</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>GIS信息</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>维度</string>
    </property>
   </widget>
   <widget class="QComboBox" name="location_comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>自动获取</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>手动输入</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="locationDisplayEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_8</zorder>
  <zorder>label_10</zorder>
  <zorder>description_textEdit</zorder>
  <zorder>size_width_lineEdit</zorder>
  <zorder>size_length_lineEdit</zorder>
  <zorder>size_height_lineEdit</zorder>
  <zorder>ok_pushButton</zorder>
  <zorder>cancle_pushButton</zorder>
  <zorder>graphicsView</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
